src/TimeSeries/encodings/timeseriespreprocessing.jl

```diff
@@ -1,3 +1,12 @@
+struct TSStats{T}
+    means::AbstractArray{T,2}
+    stds::AbstractArray{T,2}
+end
+
+function TSStats(means, stds)
+    TSStats{eltype(means)}(means, stds)
+end
```

Comment:
Not sure we need this struct, it may be simpler to add `means` and `stds` fields to the `Encoding`. Then that also makes it easier to construct `TSPreprocessing` manually (i.e. without `setup`).